### PR TITLE
fix(database): handle empty default values in column definitions

### PR DESCRIPTION
**Pull request by ki2d**
# Pull Request / 合并请求

Thank you for contributing to Kronos ORM! 请按模板填写，便于快速评审。

## Summary / 摘要
Describe what this PR does. 简述本 PR 做了什么。

## Related Issue(s) / 关联问题
- Closes #
- Relates to #

## Type of change / 变更类型
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Refactor / Optimization
- [ ] Documentation
- [ ] Test only / CI

## Affected module(s) / 影响模块
- [x] kronos-core
- [ ] kronos-logging
- [ ] kronos-jdbc-wrapper
- [ ] kronos-codegen
- [ ] kronos-compiler-plugin
- [ ] build plugin(s)

## How has this been tested? / 测试说明
- [ ] Unit tests added / updated
- [x] Manual verification
- [ ] Covers edge cases mentioned in issue

Please include:
- Kotlin version / Gradle version
- DB type(s) tested (e.g., MYSQL, POSTGRESQL)
- Logs or screenshots if relevant

```kotlin
// key test or usage snippet
```

## Documentation / 文档
- [x] Not needed
- [ ] Added/updated docs under develop-docs
- [ ] Updated README(s)

## Backward compatibility / 兼容性
- Breaking changes? If yes, describe migration path. 是否存在破坏性变更，如有请描述迁移路径。

## Checklist / 提交前检查
- [ ] I have read CONTRIBUTING.md / 我已阅读贡献指南
- [ ] I added tests that prove my fix/feature works / 我已补充或更新测试
- [ ] I updated documentation where necessary / 我已更新必要文档
- [ ] I linked related issues (e.g., "Closes #123") / 我已关联相关 Issue
- [ ] I ran detekt/formatting (if applicable) / 我已本地通过静态检查

---
References:
- CONTRIBUTING: ./CONTRIBUTING.md
- Code of Conduct: ./CODE_OF_CONDUCT.md
- Developer Docs: ./develop-docs

### CODE REVIEW

**Comment by ousc** [comment]
LGTM

File: kronos-core/src/main/kotlin/com/kotlinorm/database/sqlite/SqliteSupport.kt [hunk 1]
```diff
@@ -73,7 +73,7 @@ object SqliteSupport : DatabasesSupport {
     }${
         if (column.primaryKey == PrimaryKeyType.IDENTITY) " AUTOINCREMENT" else ""
     }${
-        if (column.defaultValue != null) " DEFAULT ${column.defaultValue}" else ""
+        if (column.defaultValue != null) " DEFAULT ${column.defaultValue.ifEmpty { "\"\"" }}" else ""
```
[the remark this answers is not in the document]
Comment (ousc) [reply kronos-core/src/main/kotlin/com/kotlinorm/database/sqlite/SqliteSupport.kt]: ok了
